entries/f/fastmail.com.json

```
@@ -9,8 +9,9 @@
       "u2f"
     ],
     "documentation": "https://www.fastmail.com/help/account/2fa.html",
+    "notes": "SMS must be set up as a backup method.",
```

Further down the doc page it says:
> Users who accept the risk of being locked out of their account may remove the recovery phone number from their account after two-step verification has been enabled.

Maybe this text (as used for other entries) is clearer:
```suggestion
    "notes": "Setup requires SMS verification.",
```